executers/dependenciesutils.go

```diff
@@ -59,6 +73,30 @@ func collectDependenciesAndPublish(targetRepo string, failOnError, publishDeps b
 	return nil
 }
 
+// Performs a head request to the deployer server to select the dependencies that need to be published to that server
+func prepareDependenciesForDeployerServer(cache *cache.DependenciesCache, dependenciesToPublish map[string]bool, resolverDeployer *params.GoResolverDeployer) error {
```

Comment:
I suggest renaming the function to *findMissingDepedencies*.

Reply:
Done